modules/doc/content/getting_started/installation/install_miniconda.md

```diff
@@ -10,20 +10,27 @@ guide.
   bash Miniforge3-Linux-x86_64.sh -b -p ~/miniforge
   ```
 
-- +Macintosh Users with Intel processors:+
-
-  ```bash
-  curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh
-  bash Miniforge3-MacOSX-x86_64.sh -b -p ~/miniforge
-  ```
-
 - +Macintosh Users with Apple Silicon processors:+
 
   ```bash
   curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh
   bash Miniforge3-MacOSX-arm64.sh -b -p ~/miniforge
   ```
 
+!alert! note title=Intel Macs nearing End of Life
+Apple will cease support for all Intel based Macintosh machines in 2026, as will we at that time.
+
+*read: As our Continuous Integration Intel machines break, they are replaced with Apple Silicon. Limiting our ability to test new features.*
```

Comment:
```suggestion
*read: As our Continuous Integration Intel machines break, they are replaced with Apple Silicon. This will limit our ability to test new features for Intel compatibility.*
```